exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -34,6 +34,42 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+func TestRejectBlankProjectID(t *testing.T) {
+	ids := []string{"", "     ", " "}
+	for _, projectID := range ids {
+		opts := Options{ProjectID: projectID}
+		exp, err := NewExporter(opts)
+		if err == nil || exp != nil {
+			t.Errorf("%q ProjectID must be rejected: NewExporter() = %v err = %q",
+				projectID, exp, err)
```

Comment:
Keep projectID, exp, and err on the previous line.

Reply:
It was a run off line, but sure, I can def update that. Thanks.